scripts/max.davinci.setup.sh

```diff
@@ -29,9 +29,9 @@ export BASE=${HOME}/degas/habanero-upc
 ################################################
 
 # COMPILERS (fixed values unless you are building on Edison)
-export CXX=mpic++
-export CXX_LINKER=mpic++
-export CC=mpicc
+export CXX=mpiicpc
```

Comment:
I don't think "mpiicpc" script would be available on all platforms. E.g., its not there on congaree, though it has mpicc